src/devices/src/virtio/block/device.rs

```diff
@@ -51,6 +51,18 @@ pub enum CacheType {
     Writeback,
 }
 
+impl CacheType {
+    /// Picks the appropriate cache type based on disk image or device path.
+    /// Special files like `/dev/rdisk*` on macOS do not support flush/sync.
+    pub fn auto(#[allow(unused_variables)] path: &str) -> CacheType {
```

Comment:
nit: could you please use `_path` instead `path` and remove the annotation. This approach is perfectly valid, but everywhere else we're using the underscore, so feels nicer for consistency.

Reply:
Sure, it's more concise too. Done.